packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_query.dart

```diff
@@ -90,20 +89,20 @@ class MethodChannelQuery extends QueryPlatform {
 
   /// Fetch the documents for this query
   @override
-  Future<QuerySnapshotPlatform> get([GetOptions /*?*/ options]) async {
+  Future<QuerySnapshotPlatform> get([GetOptions? options]) async {
     try {
-      final Map<String, dynamic> data = await MethodChannelFirebaseFirestore
+      final Map<String, dynamic>? data = await MethodChannelFirebaseFirestore
           .channel
           .invokeMapMethod<String, dynamic>(
         'Query#get',
         <String, dynamic>{
           'query': this,
           'firestore': firestore,
-          'source': getSourceString(options.source),
+          'source': getSourceString(options!.source),
```

Comment:
why is `options` optional if the first line of code in the function expects it to be non-nullable?

Reply:
Similar to your previous `GetOptions` comment, I've updated the superclass so it has the same method signature:
```dart
Future<QuerySnapshotPlatform> get([GetOptions options = const GetOptions()]) {
    throw UnimplementedError('get() is not implemented');
  }
```

Reply:
and updated the code as per your comment.

Reply:
LGTM